.github/workflows/swagger.yml

```diff
@@ -0,0 +1,29 @@
+name: Open API Specification and Documentation
+on: [push, pull_request]
+jobs:
+  swaggerci:
+    name: swagger-build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup go-swagger
+        uses: minchao/setup-go-swagger@v1
+        with:
+          version: v0.26.1
+
+  build:
+    needs: [swaggerci]
+    name: Swagger-ci
+    runs-on: ubuntu-latest
+    steps:
+    - name: swagger-build
+      uses: actions/checkout@v2

```

Comment:
This `actions/checkout@v2` is in a bad place. It should be in a single step. See:
https://github.com/meshery/meshery/blob/master/.github/workflows/ci.yml#L27